views/partials/compromised_accounts.hbs

```diff
@@ -0,0 +1,17 @@
+<div id="compromised-accounts">
+  <h3>Your Compromised Accounts</h3>
+  {{#each foundBreaches }}
+    <div class="account">
+      <div class="image-wrap">
+        <img alt="Breached Account Icon" src="img/{{ name }}.png">
+      </div>
+      <div class="breach-info-wrap">
+        <h3 class="demi">{{ name }}</h3>
+        <span class="breach-info"><span class="medium">Breach date:</span> {{prettyDate meta.BreachDate }}</span>
+        <span class="breach-info"><span class="medium">Compromised data:</span> {{breachDataClasses meta.DataClasses }}</span>
+        <span class="breach-info"><span class="medium">Compromised accounts:</span> {{localeString meta.PwnCount }}</span>
+      </div>
+    </div>
+  {{/each}}
+  <span id="source-info">Breach data provided by <span id="pwned">';--</span><span class="medium"> Have I been Pwned?</span></span>
```

Comment:
Should this be a link to their site?
Also, not sure if there is a way to hide the `';--` bit from the screenreader, but that's probably a separate a11y issue/question for the future.

Reply:
I think I asked about this and the answer was no linking, but I will double check. I'm not aware of any screenreader-only invisibility cloaks but I'll check around. Could always sub with an icon.